R/tests/testthat/test-utils.R

```diff
@@ -49,7 +49,7 @@ test_that("update_html_lines replaces path for all script lines", {
         stringr::str_detect(test_html_lines, "<script src=\"mock_chart_files/")
     )
 
-    expect_equal(test_updated_count, 5)
+    expect_true(test_updated_count %in% c(4, 5))
```

Comment:
@jaeddy can you clarify what this test is ensuring? It seems like we want to make sure all instances of `<script src=\"mock_chart_files/)` get replaced with `<script src=\"https://cdn-www.synapse.org/research/` or `<script src=\"https://ajax.googleapis.com/ajax/libs/`, is that right? I'm not sure why the counts seem to vary across runs, but if that's normal behavior then maybe instead of testing the counts we should first capture the indices of `<script src=\"mock_chart_files/)` and then ensure that each instance is changed.

Reply:
Oof. This test is bad. I feel bad.

I was getting a different test result locally vs. on Travis, thus the hacky solution. I should just fix this now.